scss/_patterns_grid.scss

```diff
@@ -19,13 +19,193 @@ $shelves-column-name: $grid-col-name;
   }
 
   // Demo helper
-  [grid-demo] [class*='col-'] {
+  [grid-demo] [class*='#{$grid-col-name}-'] {
     background: $color-mid-light;
     margin-bottom: $spv-inner--small;
   }
 
-  [grid-outline] [class*='col-'] {
+  [grid-outline] [class*='#{$grid-col-name}-'] {
     outline: 1px solid $color-x-light;
     padding: $spv-inner--small $sph-inner--small;
   }
 }
+
+@mixin vf-p-css-grid {
+  %row-reset {
+    margin-left: 0;
+    margin-right: 0;
+    max-width: $grid-max-width;
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  %span-full-grid-on-mobile {
+    @media (max-width: $breakpoint-medium) {
+      grid-column: 1 / span $grid-columns-mobile;
+    }
+  }
+
+  %span-full-grid-on-tablet {
+    @media (min-width: $breakpoint-medium) and (max-width: $breakpoint-large) {
+      grid-column: 1 / span $grid-columns-tablet;
+    }
+  }
+
+  %span-full-grid-on-desktop {
+    @media (min-width: $breakpoint-large) {
+      grid-column: 1 / span $grid-columns;
+    }
+  }
+
+  [css-grid-demo] [class*='#{$grid-column-prefix}'] {
+    background: transparentize($color-negative, 0.9);
+    margin-bottom: $spv-outer--small;
+  }
+
+  .row {
+    display: grid;
```

Comment:
We often use equal-height for example of rows which sets the row to `display: flex`. Does this pattern need reworking?

Reply:
In my tests it doesn't change that behaviour, because align-items behaves the same way in flex and css grid. Can you point me to an example where it doesn't work?

Reply:
The equal heights example seems works fine, even when the clas is on the row element: ![image](https://user-images.githubusercontent.com/2741678/56728294-5b638380-674a-11e9-9544-7fdd54db4fae.png)